src/redux/reducer/configReducer.js

```diff
@@ -56,30 +43,36 @@ const initialState = {
   nsfwWords: loadWithDefault("config.nsfwWords", []),
 };
 
-const configReducer = (state = initialState, action = {}) => {
-  switch (action.type) {
-    case "setConfigItem":
+
+const configReducer = createSlice({
```

Comment:
this usage is far cleaner, does this basically mean that `createSlice` creates the actions for us based on the reducers? 😲

Reply:
Yes